src/main/java/edu/kis/vh/nursery/Adapter.java

```diff
@@ -0,0 +1,21 @@
+package edu.kis.vh.nursery;
+
+public class Adapter implements DefaultInterface {
+	
+	protected static final int EMPTY = -1;
+	protected static final int NEW_EMPTY = 0;
+	protected static final int FULL = 11;
+	protected static final int CAPACITY = 12;
+	
+	public int getTotal() {return 0;}
+	public void countIn(int in) {}
+	public boolean callCheck() {return false;}
+	public boolean isFull() {return false;}
+	public int countOut() {return 0;}
```

Comment:
countIn, countOut and callCheck are not appropriate for implementation hierarchy (stack methods are sufficient)